webapp/src/components/open-api-plugins/plugin-wizard/steps/ValidateManifestStep.tsx

```diff
@@ -68,38 +69,33 @@ export const ValidateManifestStep: React.FC<IValidateManifestStepProps> = ({
         setErrorMessage(errorMessage);
     };
 
+    const { instance, inProgress } = useMsal();
     useEffect(() => {
         setErrorMessage(undefined);
+        AuthHelper.getSKaaSAccessToken(instance, inProgress)
+            .then(async (accessToken) => {
+                const chatService = new ChatService(process.env.REACT_APP_BACKEND_URI as string);
```

Comment:
Rec: add a hook in usePlugins that makes a call to fetch the manifest -- I think we should maintain a pattern of using the hooks to call the service and handle the request response / user action.